### ISSUE TITLE
The set up of a SQL Server database fails.

### ISSUE THREAD

**Issue by zakwillis** [mixcore/mix.core#618]
Apologies for this, I searched the issues, these kinds of things are disappointing.

It says "Server errors" when setting up a database for SQL Server. It is the first step and it falls over. :( 

Against my better judgment I;
-Tried making the user a system administrator.
- Gave a server role of create database.
- Tried using a double slash in the server\\instance and used a server\instance approach
- Creating the database.
- Not having a database. 
- Cursing at the monitor.

I will take a look and see if there is a sample DB to attach.

**Comment by zakwillis** [comment]
My fault. SQL Server wasn't in Mixed mode. May as well hang my head in shame :D

**Comment by zakwillis** [comment]
Actually, it doesn't work having changed the mode. The user connects fine. Tried creating a sql server alias to remove the instance but that can't have anything to do with it.

Will try from another machine.

**Comment by Smilefounder** [comment]
Thanks for letting us know. Is there any error screenshot that our team can refer to?

Also, are you able to connect to the same db via SQL Studio with the same connection string? And please make sure that your sql account MUST have full permission.

Which Mixcore CMS version brand or version?

**Comment by zakwillis** [comment]
hi there. I cloned from source. Just to let you know, I installed this on a separate PC with a SQL Server without an instance name. I suspect this is unrelated and instance names are fine. So I will close this. Thanks.

One thing I can say is most banks and other large enterprises wouldn't want to give SA access to an account to install a database at will. I created a SQL Server user and gave it a role of dbcreator, this was sufficient to set the database up.
![image](https://user-images.githubusercontent.com/12199408/168420008-e3dfc8b6-a844-45b2-9960-caa24dc6e629.png)

Anyway, thanks.
